src/category_theory/monoidal/tensor_product.lean

```diff
@@ -0,0 +1,82 @@
+-- Copyright (c) 2018 Michael Jendrusch. All rights reserved.
+-- Released under Apache 2.0 license as described in the file LICENSE.
+-- Authors: Michael Jendrusch, Scott Morrison
+import category_theory.products
```

Comment:
I think this file basically exists so that the definition of `monoidal_category` can go at the top of `monoidal/category`. The filename is not that illuminating, so how about adding a top-level comment
```
-- Auxiliary definitions for the definition of a monoidal category.
```

Reply:
And possible even rename the file to `monoidal_category_aux.lean` or `axioms.lean` or something like that...?

Reply:
Renamed to `category_aux.lean`, and added the comment.